SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -73,20 +73,22 @@ Worker_RequestId USpatialSender::CreateEntity(USpatialActorChannel* Channel)
 	FString ClientWorkerAttribute = GetOwnerWorkerAttribute(Actor);
 
 	WorkerAttributeSet ServerAttribute = { SpatialConstants::ServerWorkerType };
+	WorkerAttributeSet AIWorkerAttribute = { FString(TEXT("AIWorker")) };
```

Comment:
Why is this not part of the `SpatialConstants`?

Reply:
Because we need a more generic way of setting worker types than what we have here now. This code will change soon, have a note on this in the documentation I'm writing for this PR.

Reply:
I just got trolled by this - I assumed it's automatically inferred from the worker association class option for some reason...